Makefile

```diff
@@ -36,6 +43,23 @@ $(BUILD_DIR)/phenoscape-ontology-classified.ofn: $(BUILD_DIR)/phenoscape-ontolog
 	remove --term 'owl:Nothing' --trim true \
 	reason --reasoner ELK -o $@
 
+
+# Extract Qualities from ontology
+$(BUILD_DIR)/qualities.txt: $(BUILD_DIR)/phenoscape-ontology-classified.ofn
+
+
+# Extract Anatomical-Entities from ontology
+$(BUILD_DIR)/anatomical_entities.txt: $(BUILD_DIR)/phenoscape-ontology-classified.ofn
+
+
+# Create Query-Subsumers
+$(BUILD_DIR)/query-subsumers.ofn: $(BUILD_DIR)/qualities.txt $(BUILD_DIR)/anatomical_entities.txt
+
+
+# Create Similarity-Subsumers
+$(BUILD_DIR)/similarity-subsumers.ofn: $(BUILD_DIR)/qualities.txt $(BUILD_DIR)/anatomical_entities.txt
+
+# Download annotated data from Phenex
 $(BUILD_DIR)/phenoscape-data:
 	git clone https://github.com/phenoscape/phenoscape-data.git $@
```

Comment:
You might consider using the `--depth` parameter here. Given that this clone isn't going to be used for development or the traversal of commit history, it may save a lot of time (aside from disk space) downloading only a very shallow clone here.

Reply:
Thanks for this tip!